<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluriNotesClass</class>
 <widget class="QMainWindow" name="PluriNotesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PluriNotes</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Notes :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeNoteBtn">
        <property name="text">
         <string>Supprimer Note</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>To-Do List :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="taskList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(180, 180, 180);
    background-color: rgb(245, 245, 245);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Corbeille :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="bin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(180, 180, 180);
    background-color: rgb(245, 245, 245);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restoreBtn">
        <property name="text">
         <string>Retablir Note</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="NoteViewer" name="noteViewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="relationTreePanelLayout">
      <item row="2" column="0">
       <widget class="RelationTree" name="relationTree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toggleBtn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Arborescence</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextOnly</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichiers</string>
    </property>
    <widget class="QMenu" name="menuNouveau">
     <property name="title">
      <string>Nouveau</string>
     </property>
     <addaction name="actionNote"/>
    </widget>
    <addaction name="menuNouveau"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="actionRelations"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNote">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionDossier_de_stockage">
   <property name="text">
    <string>Dossier de stockage</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionRelations">
   <property name="text">
    <string>Relations..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NoteViewer</class>
   <extends>QWidget</extends>
   <header>NoteViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RelationTree</class>
   <extends>QGraphicsView</extends>
   <header>RelationTree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
